public/OutlawDerby2024.md

```diff
@@ -0,0 +1,65 @@
+![outlawlogo]("C:\Users\brian\Documents\GitHub\cub306.org\public\images\outlaw derby lOGO.png")
```

Comment:
also the direct path to your hard drive will never work on the web, you need a relative URL. The internet also sometimes does not like spaces in file names.

Reply:
im still learning markdown and how to used the editor i'm using